system/version.py

```diff
@@ -86,12 +86,7 @@ def is_prebuilt() -> bool:
 def is_comma_remote() -> bool:
   # note to fork maintainers, this is used for release metrics. please do not
   # touch this to get rid of the orange startup alert. there's better ways to do that
-  origin: Optional[str] = get_origin()
-  if origin is None:
-    return False
-
-  return origin.startswith(('git@github.com:commaai', 'https://github.com/commaai'))
-
+  return cast(str, get_normalized_origin("")) == "github.com/commaai/openpilot"
```

Comment:
why the cast here? we still want it to consider the None case

Reply:
I believe with a default value `None` won't be returned, and I have to cast to make the type checking happy because if the types on either side of `==` don't exactly match the result is `Any`.

Reply:
this throws out all the type hints and makes mypy assume it's a string. perhaps there's a bug or incomplete type hints if mypy doesn't think it's a str already

Reply:
fixed by removing default and always returning string